doc/source/data/batch_inference.rst

```diff
@@ -498,6 +497,18 @@ namely Pandas, NumPy, and Arrow, and how they're used in Ray Data:
     :start-after: __simple_numpy_start__
     :end-before: __simple_numpy_end__
 
+.. tabbed:: Pandas
+
+  The ``"pandas"`` batch format presents batches in
+  `pandas.DataFrame <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html>`__
+  format. If converting a simple dataset to Pandas DataFrame batches, a single-column
+  dataframe with the column ``"__value__"`` will be created.
```

Comment:
Remove all mention of `__value__` or "single-column" datasets.

Reply:
done